cli.js

```diff
@@ -39,15 +41,52 @@ function toHex(number, length = 32) {
 }
 
 /** Display ETH account balance */
-async function printETHBalance({ address, name }) {
-  console.log(`${name} ETH balance is`, web3.utils.fromWei(await web3.eth.getBalance(address)))
+async function printETHBalance({ address, name, symbol }) {
+  console.log(`${name} balance is`, web3.utils.fromWei(await web3.eth.getBalance(address)),`${symbol}`)
```

Comment:
Symbol for checking balance is not shown so I have added for better view ( Symbol is added automatically by netId )